client/src/Components/MetaViewer/__tests__/MetaViewer.test.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { describe, expect, test } from 'vitest';
+import { render, screen } from '@testing-library/react';
+
+import { MetaViewer, MetaViewerProps } from '../MetaViewer';
+
+describe('MetaViewer list component', () => {
+
+    const meta: MetaViewerProps = {
+        "datatype": "datatype test",
+        "version": "version test",
+        "offset": 0,
+        "sample_rate": 0,
+        "description": "description test"
+    };
+
+    test('Renders correctly', async () => {
+        render(<MetaViewer {...meta} />);
+        expect(await screen.findByText('datatype test')).toBeInTheDocument();
```

Comment:
Can we add the other values to this?

Reply:
As well as one for the truncate logic :D

Reply:
done

Reply:
We already have a truncated logic for the metadata description I think in the sigmf metadata class